templates/dice/roll-buttons.hbs

```diff
@@ -1,5 +1,5 @@
 <nav class="dialog-buttons flexrow">
     {{#each buttons}}
-    <button type="submit" data-action="{{ @key }}">{{{ icon }}} {{ label }}</button>
+    <button type="submit" data-action="{{ @key }}" {{#if default}}autofocus{{/if}}>{{{ icon }}} {{ label }}</button>
```

Comment:
Seems to work, at least on Firefox. But since we have `Dialog5e` now which handles all the dialog stuff for us, we might as well swap `RollConfigurationDialog` to be an actual dialog element.

Reply:
We can try it, we just need to be aware that it will put these roll dialogs above everything else (maybe that's desirable), and that foundry tooltips don't currently work inside `<dialog>` elements. We need to give our tooltip element the `popover` attribute so that it renders in the top layer.

Reply:
Ah, since we don't open it as a modal, these caveats don't apply.